internal/chartverifier/checks/charttesting.go

```diff
@@ -110,7 +110,7 @@ func ChartTesting(opts *CheckOptions) (Result, error) {
 	defer cancel()
 
 	cfg := buildChartTestingConfiguration(opts)
-	helm, err := tool.NewHelm(opts.HelmEnvSettings, opts.Values)
+	helm, err := tool.NewHelm(opts.HelmEnvSettings, opts.Values, opts.HelmInstallTimeout)
```

Comment:
The problem is creating a circular dependency, the tool code is set up to be independent of the rest of the chart-verifier so takes parameters as generics rather than a chart-verifier object.  One possibility is to pass parameters using the Helm structure in tools/Helm.go but not keen on that either because I prefer the calling code not to create dependencies on the tool attribute names.

Reply:
yeah i missed the circular dependency